src/Integration/WPCOMProxy.php

```diff
@@ -201,6 +245,48 @@ function ( $response, $handler, $request ) {
 		);
 	}
 
+	/**
+	 * Add the Google for WooCommerce settings to the WooCommerce REST API.
+	 *
+	 * @return void
+	 */
+	protected function add_g4w_settings() {
+		add_filter(
+			'woocommerce_settings_groups',
+			function ( $locations ) {
+				$locations[] = [
+					'id'          => self::SETTINGS_GROUP,
+					'label'       => 'Google for WooCommerce',
+					'description' => 'Settings of the Google for WooCommerce plugin.',
+				];
+				return $locations;
+			}
+		);
+
+		// Hack to make the settings group show up in the response.
```

Comment:
Hope this is robust enough to continue working. I guess in theory we are modifying a versioned API, so it should bump version if it was to break compatibility.

Reply:
I think that in the worst case if the hack stops working, the `placeholder` setting will appear in the data set, which should not be a fatal error unless API consumers will expect only responses strictly matching schema.